docs/topics/mta-web-config-jira-credentials.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-config-jira-credentials_{context}"]
+= Configuring Jira credentials
+
+You need to configure a Jira credential for establishing a connection to Jira and creating Jira issues from within the {ProductName} ({ProductShortName}) {WebName} for tracking application migrations.
+
+.Procedure
+
+. In *Administration* view, click *Credentials*.
++
+The *Credentials* page opens.
+. Click *Create new*.
+. Enter the following information:
+    * Name
+    * Description (optional)
+. In the *Type* list, select *Basic Auth (Jira)* or *Token Bearer (Jira)*:
++
+* *Basic Auth (Jira)* - used for Jira Cloud
++
+If you selected *Basic Auth (Jira)*, proceed as follows:
+
+.. In the *Email* field, enter your email.
+.. In the *Token* field, enter your token generated on the Jira site.
++
+[NOTE]
+====
+To obtain a Jira token, you need to log in to the Jira site and click *Profile* > *Personal Access Tokens* > *Create*.
```

Comment:
It depends on the version, in the latest stable version of Jira Cloud, it is under Profile -> Security -> Create and Manage API Tokens -> Create API Token

![image](https://github.com/windup/windup-documentation/assets/117646518/1c072595-129f-4159-add9-3c44dee871a3)

----

![image](https://github.com/windup/windup-documentation/assets/117646518/e92e7e7c-1278-440f-8b8e-604aa8d4f021)

Reply:
Corrected